app/controllers/movies_controller.rb

```diff
@@ -0,0 +1,6 @@
+class MoviesController < ApplicationController
+  def show
+    @movie = Movie.find(params[:id])
+    @sorted_movie_actors = @movie.actors_youngest_to_oldest
```

Comment:
Personally I would just call `@movie.actors_youngest_to_oldest` directly in the view. It wouldn't break MVC rules to do so and allows your controller to be a bit leaner.